mmv1/templates/terraform/examples/logging_folder_settings.tf.erb

```diff
@@ -0,0 +1,33 @@
+resource "google_logging_folder_settings" "<%= ctx[:primary_resource_id] %>" {
+  disable_default_sink = true
+  folder               = google_folder.my_folder.folder_id
+  kms_key_name         = google_kms_crypto_key.key.id
+  storage_location     = "us-central1"
+  depends_on           = [ google_kms_crypto_key_iam_member.iam ]
+}
+
+resource "google_kms_key_ring" "keyring" {
+  name     = "<%= ctx[:vars]['keyring_name'] %>"
+  location = "us-central1"
+}
+
+resource "google_kms_crypto_key" "key" {
+  name            = "<%= ctx[:vars]['key_name'] %>"
+  key_ring        = google_kms_key_ring.keyring.id
+  rotation_period = "100000s"
```

Comment:
To fix the VCR test failures, I think you'll need to remove these `rotation_period`s, similar to https://github.com/GoogleCloudPlatform/magic-modules/pull/7020. Basically, the VCR harness stores API request/response pairs and replays them to run the tests more quickly, and they need to be deterministic for that to work.